www/index.html

```diff
@@ -14,7 +14,7 @@
     <meta name="apple-mobile-web-app-capable" content="yes">
     <meta name="format-detection" content="telephone=no">
 
-    <script src="browserify.js" type="text/javascript" charset="utf-8"></script>
+    <script src="lib/browserify.js" type="text/javascript" charset="utf-8" concat='true'></script>
```

Comment:
What is the concat used for? I've tried to look for information about it and didn't find anything.